benchmarks/pathway.js

```diff
@@ -0,0 +1,21 @@
+var Graph  = require('../dist/src/Graph')
```

Comment:
Not for public consumption right now. Actually in a future branch I pull in `babel/register` to make the build process unnecessary.